BUILDING.md

```diff
@@ -11,13 +11,13 @@ Most people should use the [official Registry docker image](https://hub.docker.c
 
 People looking for advanced operational use cases might consider rolling their own image with a custom Dockerfile inheriting `FROM registry:2`.
 
-OS X users who want to run natively can do so following [the instructions here](https://github.com/docker/docker.github.io/blob/master/registry/recipes/osx-setup-guide.md).
+The latest updates to `main` branch are automatically pushed to [distribution Docker Hub repository](https://hub.docker.com/r/distribution/distribution) with `edge` tag.
 
 ### Gotchas
 
-You are expected to know your way around with go & git.
+You are expected to know your way around with `Go` & `git`.
```

Comment:
I get that Go is capitalised, but then putting it in a codeblock makes it look like it should be a command. Perhaps italics, but not a codeblock :)

Reply:
(Also should this and other general doc changes be in a separate pr really....)

Reply:
RE: code formatting - it was a styling change 😄 💅 I can revert

As for the general doc - I'm not sure, maybe? 🤔

Reply:
Tbh I don't mind, it can piggy back

Giving it a go before a proper review.

Reply:
Yeah, it should probably be _either_ "Go" (capitalised) or `go` (code block; the command), but not both

Reply:
Ok, made some updates to `BUILDING.md` https://github.com/distribution/distribution/pull/4069/commits/8e630ae2a5906330b41f4736f90851cf57349a04

I also wonder if we should have `DEVELOPING.md` file that contains this local environment setup instead of putting the setup guide into the `BUILDING.md`